osu.Game/Overlays/BeatmapSet/AuthorInfo.cs

```diff
@@ -49,7 +50,16 @@ private void updateDisplay()
 
             fields.Children = new Drawable[]
             {
-                new Field("mapped by", BeatmapSet.Metadata.Author.Username, OsuFont.GetFont(weight: FontWeight.Regular, italics: true)),
+                new LinkFlowContainer(s =>
```

Comment:
Please make this into its own class called `MapperLink` or similar, since it exists in two places with the same implementation. the constructor can take the font size and the metadata object.

Reply:
Sure, can do. It's actually not just 2, but 5 places now: 2 new that i implemented and 3 existing that i took for reference (linked above). I'll use just the author object since that seems a lot more intuitive